CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Unreleased
 
+## 0.8.2 (May 7, 2018)
+
+* Fix Requestable paths to prevent double slash in URI
+  
+    *Megan O'Neill*
```

Comment:
Sorry nitpick, can you remove this heading and just leave it under unreleased.

We'll add the version header in a version bump commit like: 58582e5